flake8_comprehensions.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, tree, *args, **kwargs):
         'C405': 'C405 Unnecessary {type} literal - rewrite as a set literal.',
         'C406': 'C406 Unnecessary {type} literal - rewrite as a dict literal.',
         'C407': "C407 Unnecessary list comprehension - '{func}' can take a generator.",
+        'C408': 'C408 Unnecessary {type} literal - rewrite as a tuple literal.',
+        'C409': 'C409 Unnecessary {type} literal - rewrite as a list literal.',
```

Comment:
I think that these need rewording? Perhaps:

```diff
+        'C408': 'C408 Unnecessary {type} literal passed to tuple constructor - rewrite as a tuple literal.',
+        'C409': 'C409 Unnecessary {type} literal passed to list constructor - rewrite as a list literal.',
```

Maybe a little wordy and perhaps "constructor" isn't the best term, but should be clearer...